templates/setup.cfg.tmpl

```diff
@@ -61,12 +61,12 @@ classifiers =
     Operating System :: OS Independent
 license = MIT
 license_files = LICENSE
+package_dir = {{SRC}}
 
-[option]
-package_dir=
-    ={{SRC}}
+[options]
 packages = find:
-python_requires = =={{PYTHON_VERSION}}
+python_requires = ~={{PYTHON_VERSION}}
 
 [options.packages.find]
-where = {{SRC}}
+exclude =
+    tests
```

Comment:
Why this change? 
I do not like this very much since I'd prefer to explicit list the folder(s) that should be processed in the setup process instead of letting the machine decide given an exclusion list...